docs/processes-api.md

```diff
@@ -0,0 +1,161 @@
+---
+id: processes-api
+title: Processes API
+---
+
+
+## Context
+As part of Manage My Home, there is a need to build a workflow engine that will allow housing officers and other members of staff to initiate new workflows, upload documents, assign them and capture form information. In addition, where possible, various aspects of the workflow should be automated in order to reduce the dependency on housing officers' time to carry out repeatable steps.
+
+## Vision
+- Store the state of a workflow process in order to be able to resume it at another time
+- Store the allowed next steps in the process
+- Enable assignment of a process between patches
+- Enable capture of form data
+- Allow documents to be attached to a process
+- Enable developers to quickly create and configure new and existing processes
+
+## User Needs
+As a service user, I would like to be able to:
```

Comment:
so that sections are missing

Reply:
I've added those, let me know what you think